app/register/page.tsx

```diff
@@ -1,43 +1,35 @@
 "use client";
-import RegistrationForm from "@/components/Registration/Registration";
 import Head from "next/head";
 
-import styles from "./styles.module.scss";
-import { useEffect, useState } from "react";
+import Loading from "@/components/Loading/Loading";
+import TrackSelection from "@/components/Registration/TrackSelection/TrackSelection";
 import {
     authenticate,
-    getRegistration,
     getRegistrationOrDefault,
     isAuthenticated,
-    registerUpdate,
-    registrationFromAPI,
-    registrationToAPI
+    registerUpdate
 } from "@/util/api";
-import TrackSelection from "@/components/Registration/TrackSelection/TrackSelection";
-import { RegistrationType, WithId } from "@/util/types";
-import Loading from "@/components/Loading/Loading";
+import { RegistrationType } from "@/util/types";
+import { useEffect, useState } from "react";
+import styles from "./styles.module.scss";
 
 const Registration: React.FC = () => {
-    const [isLoading, setIsLoading] = useState(true);
+    const [isLoading, setIsLoading] = useState(false);
     const [data, setData] = useState<RegistrationType | null>(null);
-    const [hasChosen, setHasChosen] = useState<boolean>(false);
 
     useEffect(() => {
         if (!isAuthenticated()) {
             authenticate(window.location.href);
             return;
         }
-
         getRegistrationOrDefault()
             .then(registration => {
                 if (registration.hasSubmitted) {
                     window.location.replace("/profile");
                 }
-
                 if ("_id" in registration) {
```

Comment:
This shouldn't exist - I realize you didn't add this but this should be removed - see main review.

Reply:
Got it. I removed this code.